compose/victimplay-predictor.yml

```diff
@@ -0,0 +1,65 @@
+# Auxiliary Docker compose file which adds services necessary for EMCTS with a predictor model for the victim.
+# Should be run in combination with compose/victimplay.yml.
+#
+# Launch command (run from repo root):
+# docker-compose -f compose/victimplay.yml -f compose/victimplay-predictor.yml --env-file compose/victimplay.env up
```

Comment:
Oh cool, I didn't realize you could chain together files like this with `docker-compose`!